geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/planning/Transforms.scala

```diff
@@ -0,0 +1,42 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.index.planning
+
+import org.geotools.process.vector.TransformProcess
+import org.geotools.process.vector.TransformProcess.Definition
+import org.locationtech.geomesa.filter.FilterHelper
+import org.opengis.feature.simple.SimpleFeatureType
+import org.opengis.filter.Filter
+
+object Transforms {
```

Comment:
Comment or rename the class:)?  What kind of transforms?!?!??!?!

Reply:
added comment